cloudferry_devlab/cloudferry_devlab/tests/mysql_connector.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2016 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+
+import sqlalchemy
```

Comment:
`sqlalchemy` seem to be too heavy for a simple SQL statement we use, I'm guessing `pymysql` library should be enough for our case.